### PR TITLE
Create reproductive organs cancer screening form

### PR DESCRIPTION
**Pull request by Jpngeta**
## Description
An implementation of a reproductive organs cancer screening form that a nurse can use to collect a patient's data.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Refactor (code changes that neither fix a bug nor add a feature)
- [ ] Other (please specify):

## Checklist

- [x] I have followed the coding standards for this project.
- [x] My code has been linted and formatted.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have updated the documentation, if necessary.
- [x] The changes require review by another team member (add reviewer).

## How Has This Been Tested?
Form Functionality Testing:

Verified all form fields render correctly
Tested conditional rendering of fields based on user input
Validated required field enforcement
Checked date field formatting and validation
Verified checkbox and radio button functionality

Data Flow Testing:

Tested form submission with various combinations of data
Confirmed proper error handling for invalid submissions

UI/UX Testing:

Checked preview mode functionality

## Screenshots or GIFs (if applicable)

https://github.com/user-attachments/assets/78c98106-f14c-45c6-a17e-041981aa1b1d

## Further Comments
The implementation includes:

Comprehensive cervical cancer screening with HPV, VIA, VIA/VILI, and Pap Smear tests
Breast cancer screening with CBE and Ultrasound tests
Conditional field rendering based on test results
Preview functionality before final submission

### CODE REVIEW

**Review by WVera123 — APPROVED** [review]
Lgtm

File: ui/src/lib/forms/validations/cancerScreeningValidation.js
Comment (1wes) [inline ui/src/lib/forms/validations/cancerScreeningValidation.js:1]: Good thinking to validate deeply nested fields this way. Well done

File: ui/src/lib/forms/cancerScreening.js
Comment (1wes) [inline ui/src/lib/forms/cancerScreening.js:1]: Lgtm

**Review by 1wes — CHANGES_REQUESTED** [review]
Remove such unnecessary commented lines and any unused variables

File: ui/src/pages/cancerScreening.jsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+      ...fieldValues.map((item) => ({ [item.name]: "" }))
+  );
+
+  const formik = useFormik({
+      initialValues: {
+          ...initialValues,
+      },
+      validationSchema: cancerScreeningValidationSchema,
+      onSubmit: (values) => {
+          //alert("clicked");
```
Comment (1wes) [inline ui/src/pages/cancerScreening.jsx]: My bad, here is an example of the commented lines of code

**Review by 1wes — CHANGES_REQUESTED** [review]

File: ui/src/pages/cancerScreening.jsx (hunk 1 above)
Comment (Jpngeta) [reply ui/src/pages/cancerScreening.jsx]: Will do

File: ui/src/lib/forms/validations/cancerScreeningValidation.js
Comment (Jpngeta) [reply ui/src/lib/forms/validations/cancerScreeningValidation.js:1]: Thanks

**Review by 1wes — COMMENTED** [review]
Awesome

**Review by 1wes — APPROVED** [review]
Thanks @Jpngeta . Merging this now

**Pull request merged**